test/conftest.py

```diff
@@ -27,6 +27,8 @@ def integrated_ff():
     integrated['ff_key'] = s3.get_access_keys()
     integrated['higlass_key'] = s3.get_higlass_key()
     integrated['ff_env'] = INTEGRATED_ENV
+    # XXX: Read from config? Somewhere else?
```

Comment:
Either a config or an environment variable (defaulting to something safe like localhost but letting you redirect), though it's best if the environment variable stuff is resolved in a config file (even if in active code like a settings.py file rather than declaratively in a `.ini` file or some such.) It's good to be able to read off all these values in one place, kind of like a style sheet (CSS) rather than litter them through the code.